### PR TITLE
Added rsvp traffic rules 

### PR DESCRIPTION
**Pull request by vvikramb**
Added routing.mpls/collect-rsvp-statson on behalf of Reinhard.

### CODE REVIEW

File: juniper_official/routing/collect-rsvp-stats.rule [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+            field lsp-stats-bytes {
+                sensor lsp {
+                    path /mpls/lsps/constrained-path/tunnels/tunnel/state/counters/bytes;
+                }
+                type integer;
+                description "LSP bytes counter";
+            }
+            field lsp-stats-counter {
+                sensor lsp {
+                    path /mpls/lsps/constrained-path/tunnels/tunnel/state/counters/name;
```
Comment (gaddevijay) [inline juniper_official/routing/collect-rsvp-stats.rule:51]: What is the significance of this? Would there be different counter names? Any documentation? or what values can come here?

Comment (gaddevijay) [reply juniper_official/routing/collect-rsvp-stats.rule:51]: Could not find these path's in telemetry explorer

File: juniper_official/routing/collect-rsvp-stats.rule [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+ *
+ */
+ healthbot {
+    topic routing.mpls {
+        rule collect-rsvp-stats {
+            keys lsp-name;
+            description "Collects LSP stats periodically";
+            sensor lsp {
+                open-config {
+                    sensor-name /junos/services/label-switched-path/usage/;
```
Comment (gaddevijay) [inline juniper_official/routing/collect-rsvp-stats.rule:21]: In already existing rsvp rule "
https://github.com/Juniper/healthbot-rules/blob/jcloud/juniper_official/routing/check-rsvp-neighbor-state.rule
" a different sensor path is used. Is there a way that we can use the sensor path for both the rules? [6:40 pm] Vijay Gadde
Or in these cases the these sensors send completely different data?

**Review by gaddevijay — CHANGES_REQUESTED** [review]

File: juniper_official/routing/collect-rsvp-stats.rule [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+ *
+ *   1) input-lsp-name, is a regular expression that matches the lsp name
+ *      that you would like to monitor.  By default it's '.*', which matches
+ *      all lsps. Use something like 'Silver*' to match only lsp names starting
+ *      with 'Silver'. 
+ *
+ */
+ healthbot {
+    topic routing.mpls {
+        rule collect-rsvp-stats {
```
Comment (gaddevijay) [inline juniper_official/routing/collect-rsvp-stats.rule:16]: What is the scale that we targeting for this rule?

**Review by gaddevijay — CHANGES_REQUESTED** [review]

File: juniper_official/routing/collect-rsvp-stats.rule (hunk 3 above)
Comment (gaddevijay) [reply juniper_official/routing/collect-rsvp-stats.rule:16]: Response from Reinhard: I haven't found any number of required LSPs to be supported. In Northstar, we have customers who have 100k + lsps to be managed, but the bulk has less than 100k.
  Having said that, I would go with 100k for now. This is not per device, but for the entire network. I believe this number will only be relevant if existing Northstar customers upgrade to EOP.

File: juniper_official/routing/collect-rsvp-stats.rule (hunk 1 above)
Comment (gaddevijay) [reply juniper_official/routing/collect-rsvp-stats.rule:51]: As per mail from Reinhard lisp-name also should be in the key

File: juniper_official/routing/collect-rsvp-stats.rule [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+                                        release-support min-supported-release;
+                                    }
+                                }
+                                platforms PTX5000 {
+                                    releases 17.4R1 {
+                                        release-support min-supported-release;
+                                    }
+                                }
+                            }
+                            products QFX {
```
Comment (gaddevijay) [inline juniper_official/routing/collect-rsvp-stats.rule]: Please remove QFX also as Reinhard only confirmed MX and PTX

File: juniper_official/routing/collect-rsvp-stats.rule [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+                                        release-support min-supported-release;
+                                    }
+                                }
+                                platforms MX960 {
+                                    releases 19.2R1 {
+                                        release-support min-supported-release;
+                                    }
+                                }
+                            }
+                            products PTX {
```
Comment (gaddevijay) [inline juniper_official/routing/collect-rsvp-stats.rule:131]: Should we add PTX in EVO?

**Review by gaddevijay — CHANGES_REQUESTED** [review]

File: juniper_official/routing/collect-rsvp-stats.rule (hunk 1 above)
Comment (vvikramb) [reply juniper_official/routing/collect-rsvp-stats.rule:51]: yes, lsp-name is key.

File: juniper_official/routing/collect-rsvp-stats.rule (hunk 4 above)
Comment (vvikramb) [reply juniper_official/routing/collect-rsvp-stats.rule]: ok, removed QFX.

File: juniper_official/routing/collect-rsvp-stats.rule (hunk 1 above)
Comment (gaddevijay) [reply juniper_official/routing/collect-rsvp-stats.rule:51]: My comment was not clear "lsp-stats-counter " should also be key

File: juniper_official/routing/collect-rsvp-stats.rule (hunk 5 above)
Comment (vvikramb) [reply juniper_official/routing/collect-rsvp-stats.rule:131]: Added PTX10001,PTX10004,PTX10008,PTX10016 for EVO.

File: juniper_official/routing/collect-rsvp-stats.rule (hunk 1 above)
Comment (vvikramb) [reply juniper_official/routing/collect-rsvp-stats.rule:51]: Added "lsp-stats-counter" also as the key.

**Review by gaddevijay — APPROVED** [review]

**Pull request merged**